test/tree/OperatorStateRetrieverUnit.tree

```diff
@@ -0,0 +1,37 @@
+.
+└── OperatorStateRetriever tree (*** denotes that integration tests are needed to validate path)
+    ├── when getOperatorState(IRegistryCoordinator,bytes32,uint32) is called
+    │   ├── when the given operator has never registered with the given registryCoordinator (their quorumBitmapHistory has length 0)
+    │   │   └── it should revert with "RegistryCoordinator.getQuorumBitmapIndicesAtBlockNumber: operator has no bitmap history at blockNumber"
+    │   └── when the given operator has registered for the first time with the given registryCoordinator
```

Comment:
what is the 'for the first time' part of this?
are you just trying to say 'has ever registered' in contrast to the above 'has never registered' ?
I have been writing these trees as assuming that if you make it down a branch, you negate any earlier branching conditions.
Ex:
-- given X
└── yadda yadda yadda (assumes *not* given X)

idk if other people have really been doing this / I appreciate explicitness though!

Reply:
ahhhh nevermind, I see what you're going for here. it's either
(a) 'the given operator registered for the first time with the given registryCoordinator after the given blockNumber'
OR
(b) 'the given operator registered for the first time with the given registryCoordinator *at or* before the given blockNumber'